### PR TITLE
PR for conference dashboard

### PR DESCRIPTION
**Pull request by LoveYourself999**
This PR is for uploading the current version of the dashboard. It uses force-direct-graph to show the connection between different keywords of conferences. The dashboard from Github Pages looks like this:
image

### CODE REVIEW

**Comment by rootfs** [comment]
@LoveYourself999 can you sign the DCO? you can follow these steps:

```
To add your Signed-off-by line to every commit in this branch:

- Ensure you have a local copy of your branch by [checking out the pull request locally via command line](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally).
- In your local branch, run: git rebase HEAD~3 --signoff
- Force push your changes to overwrite the branch: git push --force-with-lease origin gh-pages
```

File: README.md [hunk 1]
```diff
@@ -1,22 +1,28 @@
-# Cloud Native AI
```
Comment (rootfs) [inline README.md:1]: Please keep the original README.md. If you have new content, please use a different README

File: src/pages/Sidebar.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                <span className="ml-3">Dashboard</span>
+              </a>
+            </li>
+            { <li>
+              <button type="button" className="flex items-center p-2 w-full text-base font-normal text-gray-900 rounded-lg transition duration-75 group hover:bg-gray-100 dark:text-white dark:hover:bg-gray-700">
+                <span className="flex-1 ml-3 text-left whitespace-nowrap">Conference</span>
+                <svg aria-hidden="true" className="w-6 h-6" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" d="M5.293 7.293a1 1 0 011.414 0L10 10.586l3.293-3.293a1 1 0 111.414 1.414l-4 4a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414z" clip-rule="evenodd"></path></svg> 
+              </button>
+            </li>
+            /*
```
Comment (rootfs) [inline src/pages/Sidebar.js]: if this is commented out, can you remove them?

File: src/pages/api/hello.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```
Comment (rootfs) [inline src/pages/api/hello.js]: is this file needed? can you remove it?

File: src/pages/index.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import Sidebar from './Sidebar'
+import Navbar from './Navbar'
+import { useEffect, useState } from 'react';
+import ForceGraph from './force-graph';
+const Home = () => {
+  const [data, setData] = useState('');
+
+  useEffect(() => {
+    // Fetch the CSV data from the server
+    fetch('https://raw.githubusercontent.com/LoveYourself999/conference-dashboard/main/public/data/conference_data.csv')
```
Comment (rootfs) [inline src/pages/index.js]: can you update this?

File: public/data/conference_data.csv
Comment (rootfs) [inline public/data/conference_data.csv:1]: @nbcstevenchen is is ok for the summerizer to update this and other csv files?

File: src/pages/index.js (hunk 4 above)
Comment (LoveYourself999) [reply src/pages/index.js]: Sure

File: Dashboard-README.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
+# Dashboard for visualizing conference summaries
+## Github Page URL: https://loveyourself999.github.io/conference-dashboard/
```
Comment (rootfs) [inline Dashboard-README.md]: please update the URL here

File: Dashboard-README.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+Editing the page by modifying `page.js`. The page auto-updates as you edit the file.
+
+### To deploy app to Github Pages with Github Action
+First create a repository for the app, push the code.
+
+Enable GitHub Pages:
+1. Go to your repository's Settings tab
+2. Click "Pages" in the sidebar
+3. Under "Build and Deployment", select "GitHub Actions" as the source:
+<img width="1314" alt="Screen Shot 2024-04-26 at 4 57 09 PM" src="https://github.com/LoveYourself999/conference-dashboard/assets/54390035/c9999bf4-1c6c-45ee-b1e6-52d6e624b079">
```
Comment (rootfs) [inline Dashboard-README.md]: please update the URL or remove the content here

Comment (LoveYourself999) [reply Dashboard-README.md]: Should I remove the image?

Comment (rootfs) [reply Dashboard-README.md]: please add the image to the repo if it is needed to explain the overall design.
